bin/main.rb

```diff
@@ -0,0 +1 @@
+puts "Hello World"
```

Comment:
On line 1 it should be `#!/usr/bin/env ruby` for the executable main file, followed on line 2 by `puts "Hello World"`